linode_api4/groups/object_storage.py

```diff
@@ -0,0 +1,616 @@
+        :type cluster_id: str
+
+        :param bucket: The bucket name.
+        :type bucket: str
+
+        :param name: The name of the object for which to retrieve its Access Control
+                     List (ACL). Use the Object Storage Bucket Contents List endpoint
+                     to access all object names in a bucket.
+        :type name: str
+
+        :returns: The Object's canned ACL and policy.
+        :rtype: MappedObject
+        """
+        params = {
+            "name": name,
+        }
+        result = self.client.get(
+            "/object-storage/buckets/{}/{}/object-acl".format(
+                cluster_id, bucket
+            ),
+            data=drop_null_keys(params),
+        )
+
+        if not "acl" in result:
+            raise UnexpectedResponseError(
+                "Unexpected response when viewing Object’s configured ACL!",
+                json=result,
+            )
+
+        return MappedObject(**result)
+
+    def object_acl_config_update(
+        self, cluster_id, bucket, acl: ObjectStorageACL, name
+    ):
+        """
+        Update an Object’s configured Access Control List (ACL) in this Object Storage
+        bucket. ACLs define who can access your buckets and objects and specify the
+        level of access granted to those users.
+
+        This endpoint is available for convenience.
+        It is recommended that instead you use the more fully-featured S3 API directly.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-object-acl-config-update
+
+        :param cluster_id: The ID of the cluster this bucket exists in.
+        :type cluster_id: str
+
+        :param bucket: The bucket name.
+        :type bucket: str
+
+        :param acl: The Access Control Level of the bucket, as a canned ACL string.
+                    For more fine-grained control of ACLs, use the S3 API directly.
+        :type acl: str
+                   Enum: private,public-read,authenticated-read,public-read-write,custom
+
+        :param name: The name of the object for which to retrieve its Access Control
+                     List (ACL). Use the Object Storage Bucket Contents List endpoint
+                     to access all object names in a bucket.
+        :type name: str
+
+        :returns: The Object's canned ACL and policy.
+        :rtype: MappedObject
+        """
+        params = {
+            "acl": acl,
+            "name": name,
+        }
+
+        result = self.client.put(
+            "/object-storage/buckets/{}/{}/object-acl".format(
+                cluster_id, bucket
+            ),
+            data=params,
+        )
+
+        if not "acl" in result:
+            raise UnexpectedResponseError(
+                "Unexpected response when updating Object’s configured ACL!",
+                json=result,
+            )
+
+        return MappedObject(**result)
+
+    def bucket_contents(
```

Comment:
Could we also consider moving the implementation of these methods into the `ObjectStorageBucket` class? For convenience, we could also keep these functions and just make them call the underlying class methods 👍